test cases/failing/46 pkgconfig variables zero length value/meson.build

```diff
@@ -0,0 +1,16 @@
+project('variables-zero-length-test', 'c', version : '1.0')
```

Comment:
It is probably reasonable to include both of the zero length tests in the same project.